ee/vellum_ee/workflows/display/nodes/vellum/inline_prompt_node.py

```diff
@@ -15,6 +15,26 @@
 from vellum_ee.workflows.display.utils.vellum import infer_vellum_variable_type
 from vellum_ee.workflows.display.vellum import NodeInput
 
+
+def _contains_descriptors(obj):
```

Comment:
We should now be able to delete `_has_nested_descriptors` and use this method without any tests regressing

Reply:
You missed this comment